hasura.planx.uk/metadata/tables.yaml

```diff
@@ -290,6 +290,15 @@
           - id
           - version
           - data
+        validate_input:
```

Comment:
These are repeated per table, per permission.

Now that we've updated Hasura it might be worth re-looking at [inherited roles](https://hasura.io/docs/latest/auth/authorization/inherited-roles/).